source/common/upstream/resource_manager_impl.h

```diff
@@ -24,7 +24,7 @@ struct ManagedResourceImpl : public BasicResourceLimitImpl {
     remaining_.set(max);
   }
 
-  // Upstream::Resource
+  // Envoy::BasicResourceLimitImpl
```

Comment:
On second thoughts I think we normally omit the `Envoy::` namespace here

Reply:
Sure, will do